antlr/src/main/antlr/FSH.g4

```diff
@@ -27,6 +29,18 @@ csRule:             concept | caretValueRule | insertRule;
 ruleSet:            KW_RULESET SEQUENCE ruleSetRule+;
 ruleSetRule:        sdRule | concept | vsComponent;
 
+paramRuleSet:       KW_RULESET SEQUENCE PARAMETER_DEF_LIST paramRuleSetRule+;
+paramRuleSetRule:   STAR
```

Comment:
As I understand it, a `paramRuleSetRule` isn't really a single rule, but rather a block of rules. Like if the `RuleSet` was:
```
RuleSet: Foo (one, two)
* rule1
* rule2
* rule3
```
 then in this case the `paramRuleSetRule` would be all of `* rule1\n*rule 2\n*rule 3`, since the grammar only stops when it hits another keyword for an entity. Am I wrong about that and that text gets split up into three separate `paramRuleSetRule`s? If I'm not wrong, I think it would make sense to rename `paramRuleSetRule` to something like `paramRuleSetContent`, and update the grammar to indicate that there is only one block of content, not several rules.

Reply:
That's a good point. I think at one point I had `STAR` as one of the prohibited tokens, so it would actually get more than one match, but as it is written, `paramRuleSetContent` is a much more accurate name.